src/lib/components/workflow/start-workflow-button.svelte

```diff
@@ -0,0 +1,40 @@
+<script lang="ts">
+  import { twMerge as merge } from 'tailwind-merge';
+
+  import { goto } from '$app/navigation';
+
+  import Icon from '$lib/holocene/icon/icon.svelte';
+  import Tooltip from '$lib/holocene/tooltip.svelte';
+  import { routeForWorkflowStart } from '$lib/utilities/route-for';
+
+  export let namespace: string;
+  export let workflowId: string;
+  export let taskQueue: string;
+  export let workflowType: string;
+
+  let className = '';
+  export { className as class };
+
+  $: href = routeForWorkflowStart({
+    namespace,
+    workflowId,
+    taskQueue,
+    workflowType,
+  });
+</script>
+
+<Tooltip text="Start Workflow" topLeft>
+  <button
+    class={merge('start-button', className)}
+    on:click={() => goto(href)}
+    {...$$restProps}
+  >
+    <Icon title="Start Workflow" name="lightning-bolt" />
+  </button>
+</Tooltip>
+
+<style lang="postcss">
+  .start-button {
+    @apply m-1 rounded-md border-2 border-[transparent] bg-transparent p-1 hover:surface-interactive-secondary focus-visible:surface-interactive-secondary focus-visible:border-indigo-600 focus-visible:outline-none;
```

Comment:
`tailwind-merge` really doesn't like it when you mix in other classes. So, I'd recommend just putting these classes into the class string on Line 28.

It _does_ beg the question—why not just use a Holocene button?

Reply:
Remove the merge, it's not needed. It's not a Holocene button cuz those are too big for table rows. Definitely can add another variant if we want.

Reply:
We have a `size` prop on button that should do the trick. If `xs` is too small, we could add `xxs`